### PR TITLE
Add docs about amoc configuration

### PR DESCRIPTION
**Pull request by kklocek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,30 +146,34 @@ After that the scenario will keep running.
...
 - ``repeat_num`` - number of scenario repetitions for each process.
   Defaults to ``infinity``.
 
 You can also define your own entries that you might later use in your
 scenarios.
 
-The ``amoc_config`` module, that is essentially a wrapper around functions
-manipulation the application's environment in the ``application`` module.
+The ``amoc_config`` is a module for getting environment variables. For each time
+we ask for a variable, ``amoc config`` firstly looks into OS environment variables
```
Comment (michalwski) [inline README.md]: typo? should be ``amoc_config`` I guess.

File: README.md [hunk 2]
```diff
@@ -146,30 +146,34 @@ After that the scenario will keep running.
...
+(with ``AMOC_`` prefix e.g. if we want set ``interarrival`` we should set OS 
+environment variable ``AMOC_interarrival``), and if it doesn't find it gets value
+from Erlang application environment variables. What's more we can set variables 
+dynamically (via set OS variable or ``application:set_env(amoc, VARIABLE_NAME,
```
Comment (michalwski) [inline README.md]: Will the `application:set_env/3` have any effect if the `VERIABLE_NAME` is also set as OS environment variable?

Comment (kklocek) [inline README.md]: No, because OS env variables have higher priority, I will mention it in this line.

File: README.md (hunk 1 above)
Comment (kklocek) [inline README.md]: Typo, will fix it.

**Pull request merged**